src/DoctrineAuditBundle/Manager/AuditManager.php

```diff
@@ -34,6 +34,14 @@ public function getConfiguration(): AuditConfiguration
         return $this->configuration;
     }
 
+    /**
+     * @param array $payload
+     */
+    public function notify(array $payload): void
+    {
+        $this->configuration->getEventDispatcher()->dispatch(new LifecycleEvent($payload));
```

Comment:
You should check here the version of the dispatcher to send the arguments in the correct order or create your own decorated service